documentation/src/main/asciidoc/user_guide/security.adoc

```diff
@@ -101,7 +101,7 @@ There are two levels of configuration: global and per-cache. The global configur
   GlobalConfigurationBuilder global = new GlobalConfigurationBuilder();
   global
      .security()
-        .authorization()
+        .authorization().enable()
```

Comment:
@tristantarrant maybe we should enable authorization automatically when any role is set up?

Reply:
Yes, we do this kind of thing in other places where we "touch" one of the underlying props

Reply:
https://issues.jboss.org/browse/ISPN-10708